src/signals/incident-management/containers/AreaContainer/components/IncidentDetail/IncidentDetail.tsx

```diff
@@ -0,0 +1,102 @@
+import { Link } from 'react-router-dom'
+import {
+  Heading,
+  themeSpacing,
+  Link as AscLink,
+  themeColor,
+} from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+import BackLink from 'components/BackLink'
+import { string2date, string2time } from 'shared/services/string-parser'
+import type { Incident } from 'types/api/incident'
+import { isStatusEnd } from 'signals/incident-management/definitions/statusList'
+
+interface IncidentDetailProps {
+  incident: Incident
+  onBack: () => void
+}
+
+const Wrapper = styled.section`
+  padding: ${themeSpacing(4)};
+`
+
+const StyledLink = styled(AscLink)`
+  margin-bottom: ${themeSpacing(4)};
+  text-decoration: underline;
+  display: block;
+
+  :hover {
+    cursor: pointer;
+    & > * {
+      color: ${themeColor('secondary')};
+    }
+  }
+`
+
+const Section = styled.div`
+  margin-top: ${themeSpacing(4)};
+`
+
+const SectionTitle = styled.div`
+  color: ${themeColor('tint', 'level5')};
+`
+
+const Status = styled.div<{ isEnded: boolean }>`
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  color: ${(p) => themeColor('support', p.isEnded ? 'valid' : 'invalid')};
+`
+
+const IncidentDetail: React.FC<IncidentDetailProps> = ({
+  onBack,
+  incident,
+}) => {
+  const isParent = Boolean(incident._links['sia:children'])
+
+  return (
+    <Wrapper>
+      <Section>
+        <BackLink to="#" onClick={() => onBack()}>
+          {' '}
```

Comment:
Not sure if this is intentional

Reply:
nope it wasn't, my editor is very space aware